docs/src/faq.rst

```diff
@@ -0,0 +1,98 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page is a collection of FAQs. Below are links to the sections:
+
+Contents
+--------
+
+
+`Training troubleshooting`_\
+
+`General training concepts`_ \
+
+`Citing us`_ \
+
+Training troubleshooting
+------------------------
+.. _Training troubleshooting:
+
+**Q: My training fails with an out of memory error, what can I do?** \
+
+**A:** Reduce batch size.
+
+**Q: My training is very slow, what can I do?** \
+
+**A:** There are several reasons why training can be slow. If possible,
+try to reduce the dataset size or increase the batch size.
+You can also try to run on a GPU, which significantly increases performance times.
+
+**Q: My training is not converging, what can I do?** \
+
+**A:** First, please make sure that you dataset is computed consitently and converged to a reasonable accuracy.
+Looking at a distribution of your energies per atom can help. Furthermore, outliers, such as large forces
+complicate training, so looking at the distribution of the forces and removing structures with large forces
+(e.g. all structures with forces with an absolute force > 20 eV/Å) from the dataset can help to stabilize training.
+
+General training concepts
+-------------------------
+.. _General training concepts:
+
+**Q: What cutoff radius should I use?** \
+
+**A:** The optimal cutoff radius depends on the type of system you are modeling.
+
+In general, the cutoff should be large enough to include all physically relevant interactions
+(e.g., chemical bonding and short-range correlations) but not so large that it adds unnecessary
+computational cost. For istance around **4–6 Å** is a good value for most systems, but it can be
+increased to **8-10 Å** for condensed phases where longer-range effects are important. You can
+then test convergence by gradually increasing the cutoff and monitoring whether your target quantities
+(energies, forces, or other observables) change significantly.
+
+Note that if you are using massage passing the effective cutoff extends beyond the nominal atomic cutoff.
+For example, a model with a 5 Å cutoff and two message-passing layers (PET default hypers) can capture
+correlations up to roughly 10 Å.
+
+**Q: In what format should I provide my data?** \
+
+**A:** Please have a look at the getting started section for now, we will link the exact section soon.
+
+**Q: How small should my errors be before I can use my model to run Molecular Dynamics simulations?** \
+
+**A:** Good question. This depends on your system, the temperature you want to run your MD and the
```

Comment:
Maybe no need to say "good question" on our own question haha I understand why it's here but maybe it is a bit strange (?)